app/views/projects/_stage.html.erb

```diff
@@ -4,12 +4,17 @@
       <%= link_to stage.name, project_stage_path(project, stage) %>
       <%= stage_lock_icon stage %>
     </td>
-    <% if stage.last_successful_deploy %>
-      <td><%= stage.last_successful_deploy.user.name %></td>
-      <td><%= link_to stage.last_successful_deploy.short_reference, project_deploy_path(project, stage.last_successful_deploy), title: "#{stage.last_successful_deploy.reference} at #{stage.last_successful_deploy.updated_at.to_s(:db)}" %></td>
+    <% if ENV['DEPLOY_GROUP_FEATURE'] %>
+      <td><%= stage.deploy_groups.map(&:name).join(', ') %></td>
+    <% end %>
+    <% if (deploy = stage.last_successful_deploy) %>
```

Comment:
Yeah, not technically necessary. At some point I picked up the habit of adding parentheses when doing an assignment inside of a conditional statement, to have it stand out a little bit.